contracts/src/state/redeemManager/RedeemQueue.sol

```diff
@@ -15,6 +15,8 @@ library RedeemQueue {
         uint256 maxRedeemableEth;
         /// @custom:attribute The owner of the redeem request
         address owner;
```

Comment:
We should change this to `receiver` as the address represents the receiver of the redeemed ETH. It would make it more readable as well. 
Also, we should change the error `ClaimOwnerIsDenied` to `ClaimReceiverIsDenied`.

Reply:
I agree, receiver makes more sense. Would update now.